### PR TITLE
Template spec solution package update for Github -- DO NOT MERGE AS THIS IS 1P

### PR DESCRIPTION
**Pull request by v-marimanda**
This solution is packaged by making "Is1PConnector: false" as if we make this value false the CCP connector is not visible.

### CODE REVIEW

**Review by v-atulyadav — COMMENTED** [review]
Hey @NikTripathi @v-sabiraj , checks are failing please check

**Review by v-atulyadav — COMMENTED** [review]
Hi @NikTripathi @v-sabiraj, In analytical rules please add `status: Available` tag

**Review by v-atulyadav — COMMENTED** [review]
Hey @v-sabiraj, please change text from `Azure Sentinel` to `Microsoft Sentinel`

**Review by v-atulyadav — COMMENTED** [review]
Hey @v-marimanda  , please change data connector propert value from `"isPreview": true` to `"isPreview": false`

**Review by v-atulyadav — DISMISSED** [review]

**Comment by v-mchatla** [comment]
Hi @v-sabiraj
Can you please get confirmation on this PR merging or closing and act accordingly.
Thanks

**Comment by NikTripathi** [comment]
@v-sabiraj Please look into this.

File: .script/tests/detectionTemplateSchemaValidation/SkipStrcutreValidationsTemplates.json [hunk 1]
```diff
@@ -68,5 +68,17 @@
     "df0add0f-de42-4099-9657-34ae9de7a7f8",
     "9700f1da-7b1c-4702-820e-9c9ec8f2ec55",
     "d0d9aa03-561a-4855-b386-99a858259404",
-    "fa6cfcf1-b267-46d4-b348-ae7870325507"
+    "fa6cfcf1-b267-46d4-b348-ae7870325507",
+	"f4924f18-ec71-4c22-b1a9-218b6e45d992",
```
Comment (v-amolpatil) [inline .script/tests/detectionTemplateSchemaValidation/SkipStrcutreValidationsTemplates.json:72]: alignment is not correct

File: .script/tests/KqlvalidationsTests/CustomTables/GitHubAuditData.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
+{
```
Comment (v-amolpatil) [inline .script/tests/KqlvalidationsTests/CustomTables/GitHubAuditData.json:1]: is this file change required. If not remove this change

Comment (v-atulyadav) [reply .script/tests/KqlvalidationsTests/CustomTables/GitHubAuditData.json:1]: required for validation added by @v-sabiraj

**Review by v-sabiraj — APPROVED** [review]

**Pull request merged**